.travis.yml

```
@@ -0,0 +1,23 @@
+language: python
+cache: pip
+
+python:
+  - "3.7"
+
+install:
+  - pip install -r requirements-test.txt
+  - pip install coveralls
+
+script:
+  - coverage run -m pytest .
```

To limit coverage spectrum https://coverage.readthedocs.io/en/coverage-5.0.3/source.html

```suggestion
  - coverage run --source=. -m pytest
```

---

Thanks. Let's do that!

---

Done! check now :)